contrib/lib/tests/databases.rs

```diff
@@ -0,0 +1,16 @@
+#![feature(use_extern_macros)]
+
+#[macro_use] extern crate rocket;
+extern crate rocket_contrib;
+#[macro_use] extern crate log;
```

Comment:
This shouldn't be necessary.

Reply:
I removed this. It's no longer necessary.